<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="10" passed="6" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-12-24T18:18:56 IST" name="Default suite" finished-at="2020-12-24T18:19:20 IST" duration-ms="24369">
    <groups>
    </groups>
    <test started-at="2020-12-24T18:18:56 IST" name="Default test" finished-at="2020-12-24T18:19:20 IST" duration-ms="24369">
      <class name="TestClass.RequestInformationTabTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:18:56 IST" name="setup" finished-at="2020-12-24T18:18:58 IST" duration-ms="2330" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyvalidlogin()[pri:0, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:18:58 IST" name="verifyvalidlogin" finished-at="2020-12-24T18:19:08 IST" duration-ms="9385" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyvalidlogin -->
        <test-method signature="verifyclickmakebidbutton()[pri:1, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:19:08 IST" name="verifyclickmakebidbutton" description="Verify Make Bid Button" finished-at="2020-12-24T18:19:13 IST" duration-ms="5565" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyclickmakebidbutton -->
        <test-method signature="verifyclickEndStatusRowDetails()[pri:2, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:19:13 IST" name="verifyclickEndStatusRowDetails" description="Verify End status random row from the table" finished-at="2020-12-24T18:19:17 IST" duration-ms="3845" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyclickEndStatusRowDetails -->
        <test-method signature="verifyclickStartedStatusRowDetails()[pri:3, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:19:17 IST" name="verifyclickStartedStatusRowDetails" description="Verify Started status random row from the table" finished-at="2020-12-24T18:19:20 IST" duration-ms="2702" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyclickStartedStatusRowDetails -->
        <test-method signature="verifypartNameLabel()[pri:4, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:19:20 IST" name="verifypartNameLabel" description="Verify Part Name Label" finished-at="2020-12-24T18:19:20 IST" duration-ms="79" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifypartNameLabel -->
        <test-method signature="verifypartNameEnd()[pri:5, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:19:20 IST" name="verifypartNameEnd" description="Verify Part Name of End Status" finished-at="2020-12-24T18:19:20 IST" duration-ms="84" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Label is Incorrect expected:<Buffer> but was:<Test05>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Label is Incorrect expected:<Buffer> but was:<Test05>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at TestClass.RequestInformationTabTest.verifypartNameEnd(RequestInformationTabTest.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifypartNameEnd -->
        <test-method signature="verifypartNameStarted()[pri:6, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:19:20 IST" name="verifypartNameStarted" description="Verify Part Name of Started Status" finished-at="2020-12-24T18:19:20 IST" duration-ms="84" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifypartNameStarted -->
        <test-method signature="verifydescriptionLabel()[pri:7, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:19:20 IST" name="verifydescriptionLabel" description="Verify Description Label" finished-at="2020-12-24T18:19:20 IST" duration-ms="87" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Label is Incorrect expected:<Description> but was:<description>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Label is Incorrect expected:<Description> but was:<description>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at TestClass.RequestInformationTabTest.verifydescriptionLabel(RequestInformationTabTest.java:79)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifydescriptionLabel -->
        <test-method signature="verifypartDescriptionEnd()[pri:8, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:19:20 IST" name="verifypartDescriptionEnd" description="Verify Description of End Status" finished-at="2020-12-24T18:19:20 IST" duration-ms="75" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Label is Incorrect expected:<Buffer> but was:< >]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Label is Incorrect expected:<Buffer> but was:< >
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at TestClass.RequestInformationTabTest.verifypartDescriptionEnd(RequestInformationTabTest.java:88)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifypartDescriptionEnd -->
        <test-method signature="verifypartDescriptionStarted()[pri:9, instance:TestClass.RequestInformationTabTest@87a85e1]" started-at="2020-12-24T18:19:20 IST" name="verifypartDescriptionStarted" description="Verify Description of Started Status" finished-at="2020-12-24T18:19:20 IST" duration-ms="60" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Label is Incorrect expected:<Test05> but was:< >]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Label is Incorrect expected:<Test05> but was:< >
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at TestClass.RequestInformationTabTest.verifypartDescriptionStarted(RequestInformationTabTest.java:97)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifypartDescriptionStarted -->
      </class> <!-- TestClass.RequestInformationTabTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
